metricbeat/modules.d/linux.yml.disabled

```diff
@@ -10,7 +10,10 @@
     # - conntrack
     # - iostat
     # - pressure
+<<<<<<< HEAD
```

Comment:
@fearful-symmetry This and below looks a lot like something went wrong during merging. Can you clean it up in the 8.0 branch?

Reply:
Fix is already here: https://github.com/elastic/beats/pull/29326